lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -51,7 +51,7 @@ def files
         end
 
         def filters
-          engine_config.filters_for(language)
+          engine_config.filters_for(language) + default_filters
```

Comment:
What do you think of a union here, so we don't end up processing duplication patterns?

```ruby
engine_config.filters_for(language) | default_filters
```

Reply:
great suggestion

Reply:
Actually, I wonder if this works reliably with `Sexp` objects. Realizing that this aren't strings here. Worth checking on.

Reply:
Looks like this works! 👍 

```ruby
irb(main):005:0> a = Sexp::Matcher.parse("(ImportDeclaration ___)")                                                      
=> q(:ImportDeclaration, ___)                                                                                           
irb(main):006:0> b = Sexp::Matcher.parse("(ImportDeclaration ___)")                                                     
=> q(:ImportDeclaration, ___)                                                                                           
irb(main):007:0> [a] | [b]                                                                                              
=> [q(:ImportDeclaration, ___)]
```